src/masonite/routes/HTTPRoute.py

```diff
@@ -269,9 +269,11 @@ def compile_route_to_regex(self):
         return regex
 
     def extract_parameters(self, path):
+        if not self.url_list:
```

Comment:
when the route has no parameters this function was raising an error because no match found and .groups() was applied on NoneType

Reply:
good catch